config/game_options.txt

```diff
@@ -676,11 +676,11 @@ THRESHOLD_BODY_SIZE_SLOWDOWN 0.85
 
 ## Multiplier used in the smaller strides slowdown calculation.
 ## Doesn't apply to floating or crawling mobs.
-BODY_SIZE_SLOWDOWN_MULTIPLIER 0.25
+BODY_SIZE_SLOWDOWN_MULTIPLIER 3
```

Comment:
changes in config here don't affect actual server config.

Reply:
I had a feeling. I'll remove this file.

Reply:
Okay I lied, Github won't let me remove it from the PR without basically remaking the entire thing. Just pretend it's not there since it's literally changing nothing.